docs/SDK/Explanation/authentication/authSig.md

```diff
@@ -1,22 +1,14 @@
 ---
-sidebar_position: 1
+sidebar_position: 2
 ---
 
 # Wallet Signatures
 
-## AuthSigs
+An `AuthSig` is a wallet signature obtained from a user. Wallet signatures are required to communicate with the Lit Nodes and authorize requests.
```

Comment:
hmm but we have that `sign_session_key` endpoint that uses the PKP to sign an AuthSig - in this case does the first sentence still stand?

Reply:
synced with @sarahzdev - `sign_session_key` is abstracted behind walletless methods section anyways and users won't even have to think about it, so best keep this simple.